charts/posthog/Chart.yaml

```diff
@@ -11,11 +11,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 28.0.0
```

Comment:
I don't think we need to bump the major version if we don't have release notes and/or breaking changes

Reply:
It's coming here:
https://github.com/PostHog/posthog.com/pull/4562

Reply:
The "here" will link to the array "[here](https://github.com/PostHog/posthog.com/blob/f153fd92fda5dfb26617db44e40fc6ec43ef3c5b/contents/blog/the-posthog-array-1-41-0.md#1410-update-guide-for-self-hosted-users)"

Reply:
https://github.com/PostHog/posthog.com/pull/4562 WIP here